### DISCUSSION TITLE
Mixed Languages for Notifications

### DISCUSSION THREAD

**Discussion by TheRaigeki** [LazyVim/LazyVim#2140] [Q&A]
Hey there,

I have absolutely no idea where this config is - but I have a few notifications which are fully in english and a few others which are mixed in english and german.

I am using a MacBook M2 with primary language english and yeah, secondary language is german, but this should only adapt english (since this is my primary language and neovim itself is english everywhere), or am I wrong?

Does somebody had the same "problem" anytime and could help me configure this properly?
![Screenshot 2023-12-06 at 00 13 09](https://github.com/LazyVim/LazyVim/assets/39103327/8d31fe06-7ec0-44b9-9148-6d79847d93fc)
![Screenshot 2023-12-06 at 00 13 17](https://github.com/LazyVim/LazyVim/assets/39103327/5994d500-d1ee-499f-ba82-6e4fa1afa8bf)
![Screenshot 2023-12-06 at 00 13 24](https://github.com/LazyVim/LazyVim/assets/39103327/fc971e2a-ceee-40e8-841c-890194edc8d5)

**Comment by huyld** [comment]
Having the same problem now. For a long time it's just English. Now everything is in German. I can't remember what have I change that caused this.
